src/renderer/routes/test_sequencer_panel/components/SummaryTable.tsx

```diff
@@ -0,0 +1,176 @@
+import * as React from "react";
+import {
+  ColumnDef,
+  ColumnFiltersState,
+  SortingState,
+  VisibilityState,
+  flexRender,
+  getCoreRowModel,
+  getFilteredRowModel,
+  getPaginationRowModel,
+  getSortedRowModel,
+  useReactTable,
+} from "@tanstack/react-table";
+
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableHeader,
+  TableRow,
+} from "@/renderer/components/ui/table";
+import { filter, max, sum } from "lodash";
+import {
+  Summary,
+  Test,
+  TestSequenceElement,
+} from "@/renderer/types/testSequencer";
+import { useTestSequencerState } from "@/renderer/hooks/useTestSequencerState";
+
+const getOnlyTests = (data: TestSequenceElement[]): Test[] => {
+  return filter(
+    data,
+    (elem) => elem.type === "test" && elem.status != "pending",
+  ) as Test[];
+};
+
+const getCompletionTime = (data: TestSequenceElement[]) => {
+  const onlyTests = getOnlyTests(data);
+  const parallel = filter(onlyTests, (elem) => elem.runInParallel).map(
+    (elem) => elem.completionTime,
+  );
+  const non_parallel = filter(onlyTests, (elem) => !elem.runInParallel).map(
+    (elem) => elem.completionTime,
+  );
+  let max_p = max(parallel);
+  let sum_np = sum(non_parallel);
+  if (!max_p) max_p = 0;
+  if (!sum_np) sum_np = 0;
+  return max_p + sum_np;
+};
+
+const getSuccessRate = (data: TestSequenceElement[]): number => {
+  const tests = getOnlyTests(data);
+  return (
+    (filter(tests, (elem) => elem.status == "pass").length / tests.length) * 100
+  );
+};
+
+export function SummaryTable() {
+  const [sorting, setSorting] = React.useState<SortingState>([]);
```

Comment:
can you just import `useState/useEffect` from react instead of using it under the namespace just to stay consistent with other code?